ui/src/app/components/runs/filters/status-filter/status-filter.component.scss

```diff
@@ -1,6 +1,6 @@
 .status-selectable {
   cursor: pointer;
-  opacity: 0.3;
+  opacity: 0.5;
```

Comment:
Why changing the status-filter? Also, I think the status-filter should be removed, because it's not used anywhere. If we will ever need it in the future, we can probably base it on the multiple-status-filter

Reply:
Removed